examples/pipes/pipe.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: pipes.aws.upbound.io/v1beta1
```

Comment:
Could you please move the YAML file under the v1beta1 folder: `examples/pipes/v1beta1/pipe.yaml`

And please add following:
```suggestion
# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
#
# SPDX-License-Identifier: CC0-1.0

apiVersion: pipes.aws.upbound.io/v1beta1
```

Reply:
Fail on this one! Apologies about that!

Reply:
No need for apologies, thank you :)